tcms/testcases/templates/testcases/get.html

```diff
@@ -83,6 +83,30 @@ <h2 class="card-pf-title" style="text-align: left">
                 <span class="fa fa-calendar"></span>{{ object.create_date }}
             </h2>
 
+            <h2 class="card-pf-title" style="text-align: left">
+                {% with dummy_setup_duration=3000 %}
```

Comment:
Remove the `with` tag and display object properties directly. If you need real data you can update the fields directly in the DB:
```
./manage.py shell
from tcms.testcases.models import *

tc = TestCase.objects.get(pk=1234)
tc.setup_duration = timedelta(seconds=120)
tc.save()
```